chia/full_node/bitcoin_fee_estimator.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+import logging
+from datetime import datetime
+from typing import Any, Dict
+
+from chia.full_node.fee_estimate_store import FeeStore
+from chia.full_node.fee_estimation import FeeBlockInfo, FeeMempoolInfo
+from chia.full_node.fee_estimator import SmartFeeEstimator
+from chia.full_node.fee_estimator_interface import FeeEstimatorInterface
+from chia.full_node.fee_tracker import FeeTracker
+from chia.types.clvm_cost import CLVMCost
+from chia.types.fee_rate import FeeRate
+from chia.types.mempool_item import MempoolItem
+from chia.types.mojos import Mojos
+from chia.util.ints import uint32, uint64
+
+MIN_MOJO_PER_COST = 5
+
+
+def demo_fee_rate_function(cost: int, time_in_seconds: int) -> uint64:
+    return uint64(cost * MIN_MOJO_PER_COST * max((3600 - time_in_seconds), 1))
+
+
+class BitcoinFeeEstimator(FeeEstimatorInterface):
+    """
+    A Fee Estimator based on the concepts and code at:
+    https://github.com/bitcoin/bitcoin/tree/5b6f0f31fa6ce85db3fb7f9823b1bbb06161ae32/src/policy
+    """
+
+    def __init__(self, config: Dict[str, Any]) -> None:
```

Comment:
whats the reason behind using  Dict[str, Any] instead of explicit types ?

at first glance this kind of threw me off since usually a Dict[str, Any] named config in our repo means something else (the conf we are loading from file)

Reply:
Good point - I'll add a comment here

Reply:
I was using this loose type with the idea that the config sent into different estimators would contain different values.

But of course, we don't have to have this specified in `FeeEstimatorInterface`

Reply:
I switched this to being well-typed, per-estimator config params. Thank you.